frontend/src/scenes/onboarding/OnboardingReverseProxy.tsx

```diff
@@ -0,0 +1,105 @@
+    },
+    {
+        title: 'Cloudflare',
+        link: 'https://posthog.com/docs/advanced/proxy/cloudflare',
+    },
+    {
+        title: 'Kubernetes Ingress Controller',
+        link: 'https://posthog.com/docs/advanced/proxy/kubernetes-ingress-controller',
+    },
+    {
+        title: 'Netlify',
+        link: 'https://posthog.com/docs/advanced/proxy/netlify',
+    },
+    {
+        title: 'Next.js rewrites',
+        link: 'https://posthog.com/docs/advanced/proxy/nextjs',
+    },
+    {
+        title: 'Next.js middleware',
+        link: 'https://posthog.com/docs/advanced/proxy/nextjs-middleware',
+    },
+    {
+        title: 'Vercel',
+        link: 'https://posthog.com/docs/advanced/proxy/vercel',
+    },
+    {
+        title: 'Nuxt',
+        link: 'https://posthog.com/docs/advanced/proxy/nuxt',
+    },
+]
+
+export const OnboardingReverseProxy = ({ stepKey }: { stepKey: OnboardingStepKey }): JSX.Element => {
+    const { preflight } = useValues(preflightLogic)
+    const { inviteTeamMembers } = useActions(inviteLogic)
+    const { invitesToSend, canSubmit: canSubmitInvites } = useValues(inviteLogic)
+
+    return (
+        <OnboardingStep
+            title="Reverse proxy (optional)"
+            stepKey={stepKey}
+            continueText="I've already done this"
+            showSkip
+            continueAction={() =>
+                preflight?.email_service_available &&
+                invitesToSend[0]?.target_email &&
+                canSubmitInvites &&
+                inviteTeamMembers()
+            }
+        >
+            <div className="mb-6 mt-6">
+                <p>A reverse proxy allows you to send events to PostHog Cloud using your own domain.</p>
+                <p>
+                    This means that events are sent from your own domain and are less likely to be intercepted by
+                    tracking blockers. You'll be able to capture more usage data without having to self-host PostHog.
+                </p>
+                <p>
+                    Setting up a reverse proxy means setting up a service to redirect requests from a subdomain you
+                    choose (like <span className="font-mono break-keep">e.yourdomain.com</span>) to PostHog. It is best
+                    practice to use a subdomain that does not include posthog, analytics, tracking, or other similar
+                    words.
+                </p>
+                <h3>Documentation</h3>
+                <p>Here are some popular reverse proxy options:</p>
+                <ul className="list-disc list-inside ml-2">
+                    {proxyDocs.map(({ title, link }) => (
+                        <li key={title}>
+                            <Link to={link} target="_blank">
+                                {title}
+                            </Link>
+                        </li>
+                    ))}
+                </ul>
+                <LemonCard className="mt-6" hoverEffect={false}>
```

Comment:
I think this card looks too obvious and draws attention away from the main body of this step. 

It might be sufficient to throw a divider below the main text and then have this div (no bg color) below the divider. 

Another idea would be to have a third button in the bottom right for "need help?" that opens the invite modal .

A THIRD and super _subtle_ idea would be to hide the modal in a hoverable question mark to the right of the "reverse proxy (optional)" title. That might also be totally at odds with what people expect the question mark to be for and confuse users.

Reply:
That's a good call. I like the separator + no bg idea. I'll make that change.